### PR TITLE
issue/3252-order-list-search-shadow

### LINKED ISSUE #3252: Orders: Search field should show selected filter
**Describe the bug**

As @Garance91540 reported:

Once I select a search criteria (ie “Pending payment”), this is the screen I see but the selected filter should appear in the search bar (right now the search bar is empty)

**To Reproduce**
1. Go to Orders
2. Select the search icon
3. Select a filter on the search screen
4. Notice the search bar is empty

**Screenshots**

**Mobile Environment**
Please include:
- Device: 
- Android version: 
- WooCommerce Android version: 5.6-rc-1

### PR DESCRIPTION
**Pull request by nbradbury**
Fixes #3252 by adding/removing the tab layout when the order list search view collapses/expands. This avoids showing a shadow under the toolbar when searching orders.

All credit to @hichamboushaba for this solution.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt [hunk 1]
```diff
@@ -573,6 +573,7 @@ class OrderListFragment : TopLevelFragment(),
         showTabs(false)
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt]: Nitpick: I think this call is redundant as we are removing the tabLayout later on, and I think we can remove the `showTabs(true)` from the method `disableSearchListeners` as well.

**Review by hichamboushaba — DISMISSED** [review]
LGTM, I left one nitpick comment, you can merge the PR after you check it.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt (hunk 1 above)
Comment (nbradbury) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt]: Good call - addressed in 2d31e3d.

**Comment by hichamboushaba** [comment]
@nbradbury there some conflicts that need to be fixed before merging this one.

**Comment by nbradbury** [comment]
> there some conflicts that need to be fixed before merging this one.

Don't you love it when one of your branches gets conflicts due to another of your branches being merged? :)

Fixed in 38a1034.

**Review by hichamboushaba — APPROVED** [review]
:shipit:

**Pull request merged**